main/solution/backend/config/settings/.defaults.yml

```diff
@@ -190,6 +203,12 @@ dbRoleAllocations: ${self:custom.settings.dbPrefix}-RoleAllocations
 # DynamoDB table name for Resource usages
 dbResourceUsages: ${self:custom.settings.dbPrefix}-ResourceUsages
 
+# DynamoDB table name for Egress Store
+dbEgressStore: ${self:custom.settings.dbPrefix}-EgressStore
+
+# DynamoDB table global secondary index name for the "WorkspaceIdIndex" field
+dbEgressStoreWorkspaceIdIndex: WorkspaceIdIndex
+
```

Comment:
For some of these properties, isn't it best to manage and name them internally. Any reason we would want the admin to have control of index and table names?

Reply:
By extracting the properties, we are offering more flexibility to customers in their Path to Production and they may edit those field without dive deep into CloudFormation details. Since those properties are added in defaults YAML file, we are not putting extra burden to customers who are going to deploy the solution. They just need to focus on <Stage>.yml file in most cases.

Reply:
We spoke offline. As you pointed out these settings are not part of the main config so keeping it here makes sense. We should just document that changing these settings can have undesirable consequences on egress feature so it isn't recommended that we change them. Looks good otherwise. Sorry about the unnecessary churn.